beacon_node/client/src/builder.rs

```diff
@@ -471,6 +472,7 @@ where
     }
 
     /// Starts the networking stack.
+    #[instrument(name = "start_networking_stack", skip_all)]
```

Comment:
I'm having trouble getting this span and the root span `start_beacon_node` exported to OTLP backend (tempo), and I couldn't figure out why. I don't see any errors in Tempo, so I suspect it's getting filtered / errorred while exporting.